samples/KafkaAvroSample/com.ibm.streamsx.kafka.sample/KafkaAvroSample.spl

```diff
@@ -0,0 +1,76 @@
+namespace com.ibm.streamsx.kafka.sample;
+
+use com.ibm.streamsx.kafka::KafkaConsumer;
+use com.ibm.streamsx.kafka::KafkaProducer;
+use com.ibm.streamsx.avro::TupleToAvro;
+use com.ibm.streamsx.avro::AvroToJSON;
+
+/*
+ * This sample uses AVRO blob for message.
+ */
+composite KafkaAvroSample
+{
+	type
+		Message = blob message, rstring key;
```

Comment:
Don't define a message type here. Use `MessageType.BlobMessage` for the schema:
```
use com.ibm.streamsx.kafka::MessageType;
// messages get de-serialized into blob
stream <MessageType.BlobMessage> ReceivedMessages = KafkaConsumer() {
    ...
}
```
The same schema can be used for the stream consumed by the KafkaProducer.

Reply:
Assuming you are doing the change, I'm going to approve the PR.